website/docs/users-sources/sources/protocols/kerberos/index.md

```diff
@@ -0,0 +1,130 @@
+
+You need to create a principal for authentik and a keytab:
+
+```bash
+$ kadmin
+> add_principal HTTP/authentik.company@REALM.COMPANY
+> ktadd -k /tmp/authentik.keytab HTTP/authentik.company@REALM.COMPANY
+> exit
+$ cat /tmp/authentik.keytab | base64
+$ rm /tmp/authentik.keytab
+```
+
+In authentik, you need to configure these extra options:
+
+-   SPNEGO keytab: the base64-encoded keytab created above.
+
+If you do not wish to use a keytab, you can also configure authentik to use an existing credentials cache.
+
+You can also override the SPNEGO server name if needed.
+
+You may need to configure your web browser to allow SPNEGO. Check out [our documentation](./browser.md) on how to do so. You can now login to authentik using SPNEGO.
+
+### Custom server name
+
+If your authentik instance is accessed from multiple domains, you may want to force the use of a specific server name. You can do so with this option. It must be in the form of `HTTP@authentik.company`.
+
+If not specified, this defaults to trying out all entries in the keytab/credentials cache until a valid server name is found.
+
+## Extra settings
+
+There are some extra settings you can configure:
+
+-   Update internal password on login: when a user logs in to authentik using the Kerberos source as a password backend, their internal authentik password will be updated to match the one from Kerberos.
+-   Use password writeback: when a user changes their password in authentik, their Kerberos password will be updated to match the one from authentik. This is only available if synchronization is configured.
+
+## Kerberos source property mappings
+
+See the [overview](../../property-mappings/index.md) for information on how property mappings work.
+
+By default, authentik ships with [pre-configured mappings](#built-in-property-mappings) for the most common Kerberos setups. These mappings can be found on the Kerberos Source Configuration page in the Admin interface.
+
+### Built-in property mappings
+
+Kerberos property mappings are used when you define a Kerberos source. These mappings define which Kerberos property maps to which authentik property. By default, the following mappings are created:
+
+-   authentik default Kerberos User Mapping: Add realm as group
+    The realm of the user will be added as a group for that user.
+-   authentik default Kerberos User Mapping: Ignore other realms
+    Other realms than the one configured on the source will be ignored, and not be allowed to log in.
+-   authentik default Kerberos User Mapping: Ignore system principals
+    System principals such as `K/M` or `kadmin/admin` are ignored.
+-   authentik default Kerberos User Mapping: Multipart principals as service accounts
+    Multipart principals (for example: `HTTP/authentik.company`) have their user type set to service account.
+
+These are configured with the most common Kerberos setups.
+
+### Expression data
+
+The following variables are available to Kerberos source property mappings:
```

Comment:
Because of the heading title, I am guessing these variables should be added to a config file... but can we make it more clear where exactly they add this as an expression?

Also, is it plural? I only see the one variable, `principal`.

Reply:
I just took whatever we have in other sources. In the future, we might add more

Reply:
OK, then let's make it singular for now.. and we can pluralize it later when we add another variable.